locale/ru_RU/countries.xml

```diff
@@ -35,7 +32,7 @@
 	<country name="Бахрейн" code="BH"/>
 	<country name="Бангладеш" code="BD"/>
 	<country name="Барбадос" code="BB"/>
-	<country name="Беларусь" code="BY"/>
+	<country name="Белоруссия" code="BY"/>
```

Comment:
Thanks, @ppv1979 -- Frankly the issue is that I'm ill-informed about geopolitics and have been bitten by it before :) Basically, as long as we're following ISO 3166-1 conventions I'm happy. Obviously users are free to adapt their installations if they're not satisfied with that but I generally try to take the least controversial approach by default.